var/spack/repos/builtin/packages/cube-blade/package.py

```diff
@@ -10,14 +10,15 @@ class CubeBlade(AutotoolsPackage):
     """Simple OTF2 trace explorer"""
 
     homepage = "https://www.scalasca.org/software/cube-4.x/download.html"
-    url = "https://apps.fz-juelich.de/scalasca/releases/cube/4.5/dist/blade-0.2.tar.gz"
+    url = "https://apps.fz-juelich.de/scalasca/releases/cube/4.8/dist/blade-0.5.tar.gz"
+    maintainers("swat-jsc")
 
-    version("0.2", sha256="ab3c5bbca79e2ec599166e75b3c96a8f6a18b3064414fc39e56f78aaae9c165c")
+    version("0.5", sha256="0770cd4c2e1b8f31bdb6dadf39232b709aee869835e12f9e1ac670d0b276a689")
```

Comment:
Normally newer versions are added and old ones are deprecated.  There are exceptions so check out the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions to see if that applies to this package.

Reply:
After checking, I cannot fetch `0.2` so deprecation for one Spack release appears to be in order in case it is in someone's binary cache.

If you choose that route, then you'll likely need conditional dependencies below (see docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#conditional-dependencies).